src/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -16,12 +16,10 @@ namespace System.Net.Security.Tests
 
     public class SslStreamNetworkStreamTest
     {
-
         [Fact]
-        [ActiveIssue(16516, TestPlatforms.Windows)]
-        public async void SslStream_SendReceiveOverNetworkStream_Ok()
+        public async Task SslStream_SendReceiveOverNetworkStream_Ok()
         {
-            TcpListener listener = new TcpListener(IPAddress.Any, 0);
+            TcpListener listener = new TcpListener(IPAddress.Loopback, 0);
```

Comment:
If I added this test, I probably just fat-fingered this with IntelliSense, as I would have intended Loopback in the first place, so good change :)